### PR TITLE
Create BLadd26553.xml

### PR DESCRIPTION
**Pull request by DariaElagina**
Do you have an idea who this Rich from a1 is?

### CODE REVIEW

File: LondonBritishLibrary/add/BLadd26553.xml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                                <desc>Neatly written</desc>
+                            </handNote>
+
+                        </handDesc>
+
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#1v"/>
+                                    <desc type="ScribalSignature"></desc>
```
Comment (thea-m) [inline LondonBritishLibrary/add/BLadd26553.xml:103]: I am not sure from the description whether it is a scribal signature, we do not seem to know whether this note was written by Rich or someone else.
I don't know who he is either.

**Comment by DariaElagina** [comment]
@thea-m , thank you!

File: LondonBritishLibrary/add/BLadd26553.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/add/BLadd26553.xml]: I would assume that the entire text of this small manuscript might have transcribed by Mr Rich in his early years, 
and he left a note about this.

Comment (DariaElagina) [reply LondonBritishLibrary/add/BLadd26553.xml]: I will leave the category Unclear untill we could say more about the character of this note.

**Review by thea-m — APPROVED** [review]

**Pull request merged**